config/dependencies/istio/sail/istio.yaml

```diff
@@ -1,15 +1,12 @@
-apiVersion: operator.istio.io/v1alpha1
+apiVersion: sailoperator.io/v1alpha1
 kind: Istio
 metadata:
   name: default
 spec:
-  version: v1.20.0
+  # Supported values v1.22.4|v1.23.0
+  version: v1.22.4
```

Comment:
Worth noting that this release of the sail operator only supports newer versions of Istio than the v1.20.0 that we are on for the `istioctl` install

Reply:
In fact there seems to be an issue with `v1.22.4` on the 0.1.0 release which was not present in the rc. The default profile is missing. This works with `v1.23.0` however so I've updated in the latest commit